pages/common/pdflatex.md

```diff
@@ -1,7 +1,7 @@
 # pdflatex
 
 > Compile a PDF document from LaTeX source files.
-> More information: <https://linux.die.net/man/1/pdflatex>.
+> More information: <https://www.unix.com/man-page/redhat/1/pdflatex>.
```

Comment:
I saw your commend above about this, and I think we can point this to Arch man pages even if the Arch link says `pdftex`. After all, even this page documents `pdftex`, so I don't see a problem.